client/src/App.css

```diff
@@ -1,3 +1,51 @@
+#root {
+  background-color: #181a1b;
+}
 .App {
   text-align: center;
+  color: rgb(232, 230, 227);
+  padding: 1em;
+}
+
+.logo {
+  width: 5em;
+}
+
+.addVideoContainer {
+  text-align: center;
+}
+
+.card-group {
+  display: flex;
+  flex-flow: row wrap;
+  justify-content: space-around;
+}
+.card {
```

Comment:
your iframe is overhanging the card at the mo, if you set the width of the iframe to 100% should fit nicer :)